webservices/schemas.py

```diff
@@ -250,7 +250,6 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
     },
     options={
         'exclude': (
-            'memo_code',
```

Comment:
It was included there because it was in there under a different name, I think we will want tok phase out that name too. I can add that to the email.